src/utils/mapUtils.ts

```diff
@@ -27,7 +28,7 @@ export const findBounds = (
   coords: number[][],
   paddingBottom = 250,
 ): MapBounds | undefined => {
-  if (coords.length === 0) {
+  if (coords.length === 0 || !isNumber(coords[0]) || !isNumber(coords[1])) {
```

Comment:
Wouldn't both `coords[0]` and `coords[1]` be arrays here?

Reply:
Good catch, moved the check down to check the actual numbers